zarr/tests/test_convenience.py

```diff
@@ -587,7 +587,7 @@ def temp_h5f():
     return h5f
 
 
-@pytest.mark.skipif(h5py is None, reason='h5py not installed')
+@unittest.skipIf(h5py is None, 'h5py is not installed')
```

Comment:
Not urgent, but may want to give [`pytest.importorskip`]( https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency ) a look.

Reply:
Should add this would go inside the function.

Reply:
That's nice, I guess it could go in a setUp() method for each relevant class. I think I'll leave for now but will raise an issue to revisit this.